src/test/kotlin/racingcar/view/InputViewTest.kt

```diff
@@ -0,0 +1,30 @@
+package racingcar.view
+
+import io.kotest.core.spec.style.AnnotationSpec
+import io.kotest.matchers.shouldBe
+import racingcar.view.input.Input
+
+class InputViewTest : AnnotationSpec() {
+    @Test
+    fun `Input 의 결과를 보관한다`() {
+        val expectCarCount = 5
+        val expectTryCount = 3
+        val input = StubInput(expectCarCount, expectTryCount)
+        val sut = InputView(input)
+
+        sut.start()
+
+        sut.carCount shouldBe expectCarCount
+        sut.tryCount shouldBe expectTryCount
+    }
+}
+
+class StubInput(var carCount: Int, var tryCount: Int) : Input() {
```

Comment:
기본적으로 `val` 키워드를 사용하여 모든 변수를 읽기 전용으로 선언하고 필요할 때만 `var`로 변경하면 어떨까요?

```suggestion
class StubInput(val carCount: Int, val tryCount: Int) : Input() {
```

Reply:
앗? val을 안썼군요! 확인 감사합니다!